PyInstaller/hooks/hook-zmq.py

```diff
@@ -17,18 +17,19 @@
 
 hiddenimports = ['zmq.utils.garbage'] + collect_submodules('zmq.backend')
 
-# If PyZMQ provides its own copy of libzmq and libsodium, add it to the
-# extension-modules TOC so zmq/__init__.py can load it at runtime.
-# PyZMQ is able to load 'libzmq' and 'libsodium' even from sys._MEIPASS.
-# So they could be with other .dlls.
-try:
-    binaries = [(get_module_file_attribute('zmq.libzmq'),
-                 '.' if is_py2 and is_win else 'zmq')]
-except ImportError:
-    # Not all platforms provide their own copy of libzmq.
-    pass
-else:
-    # If PyZMQ pvorides its own copy of libzmq and libsodium, these libs look like
-    # C extensions. Excluding these modules ensures that those dlls are not bundled
-    # twice. Once as ./zmq.libzmq.pyd and once as ./zmq/libzmq.py.
-    excludedimports = ['zmq.libzmq']
+if is_py2:
```

Comment:
Please add a comment, why this is only required for Py2.

Reply:
Done.